browser/app/js/components/Browse.js

```diff
@@ -68,7 +68,7 @@ export default class Browse extends React.Component {
           memory: res.MinioMemory,
           platform: res.MinioPlatform,
           runtime: res.MinioRuntime,
-          envVars: res.MinioEnvVars
+          info: res.MinioGlobalInfo
```

Comment:
@harshavardhana i will be closing this PR and opening a new one with the same changes (without the confusion we have in this PR due to an incorrect rebase with master branch).